src/user-guide/installing-workflows.rst

```diff
@@ -402,7 +402,7 @@ The run directory now looks as follows:
 
    If your workflow needs to create or install scripts or executables at runtime
    and you don't want Cylc to delete them on re-installation, you can use
-   ``bin`` and ``lib/python`` directories in the :ref:` workflow share directory <workflow_share_directories>`.
+   ``bin`` and ``lib/python`` directories in the :ref:`workflow_share_directories`.
```

Comment:
?

Reply:
This link was broken. Dave spotted it whilst doing my review.

Reply:
Eh? This change didn't change the link, it just removed the placeholder text "workflow share directory".

Reply:
Didn't appear to - have a look at it on the compiled docs...

Reply:
I'm not doubting the link was broken (I'm sure you tested it), but the change shouldn't have fixed it because the target is the same between the two examples.

Taking a look above, you can see there colour of the `:ref:` is different before and after so there must have been a syntax error, looks like it's the space:

```rst
:ref:` workflow share directory <workflow_share_directories>`
:ref:`workflow share directory <workflow_share_directories>`
```

So this should do it:

```suggestion
   ``bin`` and ``lib/python`` directories in the :ref:`workflow share directory <workflow_share_directories>`.
```

Reply:
But the placeholder test is unecessary

Reply:
Yes and no.

Your diff would change the text like so:

```diff
- ``bin`` and ``lib/python`` directories in the workflow share directory
+ ``bin`` and ``lib/python`` directories in the Workflow Share Directory
```

If you don't specify the placeholder text, then it defaults to the title of the section. This is a useful default, but there are two issues:

1. It can lead to problems when sections are renamed.
2. The capitalisation is typically unsuitable.

Reply:
> Someone took the time to add a placeholder here, we should preserve it.

Me, about a fortnight ago

Reply:
```suggestion
   ``bin`` and ``lib/python`` directories in the :ref:`workflow share directory <workflow_share_directories>`.
```